docs/import-image.md

```diff
@@ -25,9 +25,12 @@ To import virtual machine images in the **Images** page, enter a URL that can be
 Currently, qcow2, raw, and ISO images are supported.
 
 !!!note
-    Please do not refresh the page until the file upload is finished.
+
+    - Please do not refresh the page until the file upload is finished.
+    - This feature is temporarily unavailable, [a known issue](https://github.com/harvester/harvester/issues/1415).
```

Comment:
This feature is temporarily unusable on the UI, a known issue will be fixed via [#1415](https://github.com/harvester/harvester/issues/1415).